src/network/networkd-ndisc.c

```diff
@@ -201,6 +202,12 @@ static int ndisc_request_route(Route *in, Link *link, sd_ndisc_router *rt) {
                         return log_link_warning_errno(link, r, "Failed to get default router hop limit from RA: %m");
         }
 
+        if (link->network->ipv6_accept_ra_use_retrans_time) {
+                r = sd_ndisc_router_get_retrans_time(rt, &retrans_time);
```

Comment:
you query the value here, but don't use it anywhere?

Reply:
> you query the value here, but don't use it anywhere?

I don't believe I need it, so will remove the code.

After more code reading and learning , I believe this code path is used to update the `Route` structure which is later used in `networkd-route.c` to write to the kernel via netlink using the `RTAX_*` members. I don't believe there is any need to update the kernel in this way with the `retrans_time`, so I'm removing this code.  Let me know if you feel I'm overlooking something.